src/games/strategy/triplea/util/TransportUtils.java

```diff
@@ -22,11 +21,7 @@
 public class TransportUtils {
```

Comment:
The diff of this class is pretty much unreadable do to several methods being refactored and moved around. Suggest either looking over the final class or review commit by commit.